.github/CODEOWNERS

```diff
@@ -1,30 +1,16 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-# More details are here: https://help.github.com/articles/about-codeowners/
-
-# The '*' pattern is global owners.
-
-# Order is important. The last matching pattern has the most precedence.
-# The folders are ordered as follows:
-
-# In each subsection folders are ordered first by depth, then alphabetically.
-# This should make it easy to add new rules without breaking existing ones.
-
-# Global rule:
-*                            @janus-idp/maintainers-plugins
-yarn.lock                    @janus-idp/maintainers-plugins
-*/yarn.lock                  @janus-idp/maintainers-plugins
-
-# Documentation:
-*.md                         @janus-idp/maintainers-docs
-
-# Plugin specific owners:
-/plugins/jfrog-artifactory/  @janus-idp/maintainers-plugins @fmenesesg
-/plugins/3scale-backend/     @janus-idp/maintainers-plugins @fmenesesg
-/plugins/quay/               @janus-idp/maintainers-plugins @fmenesesg
-/plugins/topology/           @janus-idp/maintainers-plugins @debsmita1 @divyanshiGupta
-/plugins/tekton/             @janus-idp/maintainers-plugins @debsmita1 @divyanshiGupta
-/plugins/rbac-backend/       @janus-idp/maintainers-plugins @gorkem @AndrienkoAleksandr @PatAKnight
-/plugins/rbac-common/        @janus-idp/maintainers-plugins @gorkem @AndrienkoAleksandr @PatAKnight
-
+*                           @backstage/maintainers
+*.md                        @janus-idp/maintainers-docs
+*/yarn.lock                 @janus-idp/maintainers-plugins
+/plugins/3scale-backend/    @janus-idp/maintainers-plugins @fmenesesg
+/plugins/feedback           @yashoswalyo
+/plugins/feedback-backend   @yashoswalyo
```

Comment:
will it just be one owner or there will be any co-owners?

Reply:
Sorry, there would be co-owners too for the plugin

Reply:
I have updated the CODEOWNERS file with the changes